src/framework/analyses.ml

```diff
@@ -392,22 +374,19 @@ sig
 end
 
 type analyzed_data = {
-  cil_file: Cil.file ;
   solver_data: Obj.t;
 }
 
 type increment_data = {
   server: bool;
 
   old_data: analyzed_data option;
-  new_file: Cil.file;
   changes: CompareCIL.change_info
 }
 
-let empty_increment_data ?(server=false) file = {
+let empty_increment_data ?(server=false) () = {
   server;
   old_data = None;
```

Comment:
It's interesting that the file need not be saved here...

Reply:
The file isn't saved in the incremental data, but with a separate variant into a separate file using `Serialize`, so this field was never read from. Actually #357 would probably change this and put the file (along with everything else) into a single record and not have separate files. Depending on the choices taken for #357, it need not be this increment data record that everything goes into, but maybe some outer wrapping record.